src/main/java/com/google/cloud/tools/managedcloudsdk/install/Installer.java

```diff
@@ -61,10 +61,8 @@ public void install()
     command.add("--quiet"); // don't accept user input during install
     command.add("--usage-reporting=" + usageReporting); // usage reporing passthrough
 
-    progressListener.start("Installing Cloud SDK", 2);
-    progressListener.update(1);
+    progressListener.start("Installing Cloud SDK", ProgressListener.UNKNOWN);
```

Comment:
It looks like this progressListener isn't being passed into anything or updated? Is this just going to be a 0/100 progress?

Reply:
Yeah. We can explore adding a watcher to the console, and maybe add an update(1) for every read we make from the console, but I don't know about doing that right now.